app/controllers/interests_controller.rb

```diff
@@ -1,6 +1,7 @@
 class InterestsController < ApplicationController
   def new
     vacancy = Vacancy.find(vacancy_id)
+    VacancyGetMoreInfoClick.new(vacancy).track
```

Comment:
It's not on the card but do you think we should be consistent with the page view counter and filter out authenticated users who click 'get more info'?

Reply:
Yeah, good call